README.md

```diff
@@ -51,43 +51,31 @@ pip3 install -r requirements.txt
 npm install
 ```
 
-**Add a development config file inside the config folder: `./config/config.development.json` With the same structure as in `./config/config.sample.json`**
```

Comment:
let's not remove this README part, it is required for local development

Reply:
I didn't remove it, I just add rows : bash export ENV="production"